src/arch/InputHandler/LinuxInputManager.h

```diff
@@ -18,10 +25,13 @@ class LinuxInputManager
 	void InitDriver(InputHandler_Linux_Event* drv);
 	~LinuxInputManager();
 private:
+	std::vector<LinuxInputSort> m_vPreSort;

```

Comment:
this should be local to `LinuxInputManager::PresortPhysical`, not a member variable